app/models/user.rb

```diff
@@ -2,6 +2,7 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
+require 'zip'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.